x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldMapper.java

```diff
@@ -198,8 +221,12 @@ public GeoShapeWithDocValuesFieldType fieldType() {
     }
 
     @Override
-    protected String contentType() {
-        return CONTENT_TYPE;
+    protected void addStoredFields(ParseContext context, Geometry geometry) {
+
     }
 
+    @Override
+    protected void addMultiFields(ParseContext context, Geometry geometry) {
+
```

Comment:
It looks like the old code had a comment that this was intentionally a noop. I think it is worth keeping that.